docs/docs/getting-started/first-steps.md

```diff
@@ -164,6 +164,35 @@ If you don't have a cloud subscription, check out [MiniConstellation](first-step
 
         Supported are all machines from the N2D family. It defaults to `n2d-standard-4` (4 vCPUs, 16 GB RAM), but you can use any other VMs from the same family. Refer to [N2D machine series](https://cloud.google.com/compute/docs/general-purpose-machines#n2d_machines) or run `constellation config instance-types` to get the list of all supported options.
 
+    </tabItem>
+    <tabItem value="aws" label="AWS">
+
+    * **region**: The name of your chosen AWS data center region, e.g., `us-east-2`.
+
+        TODO: What is the list of supported regions for our images?
```

Comment:
@malt3 could you help me out with that point? `config generate` does not fill that value in. I see that this is not a static value as with GCP or Azure, should we add some logic to CLI to fetch this value based on the region? Is a manual approach currently preferred?